sonar-scanner-engine/src/main/java/org/sonar/batch/source/DeprecatedDefaultSymbolTable.java

```diff
@@ -70,6 +77,11 @@ public void newReference(Symbol symbol, int fromOffset, int toOffset) {
     }
 
     @Override
+    public void newReference(Symbol symbol, int startLine, int startLineOffset, int endLine, int endLineOffset) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Symbol"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)